site/pages/party.th.rst

```diff
@@ -0,0 +1,137 @@
+.. title: PyCon Party
+.. slug: party
+.. date: 2018-06-04 20:23:32 UTC+07:00
+.. tags: 
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+
+.. |mdash| unicode:: U+02014 .. EM DASH
+.. role:: small-caps
+
+PyCon Party
+=================
+
+ชวนทุกท่านมาร่วมงานปาร์ตี้ หลังจาก Talk สุดท้ายจบลงในคืนวันเสาร์! 
+
+เรามีพิซซ่าและเครื่องดื่ม ในเวลาหนึ่งทุ่มครึ่ง ( ฟรีเครื่องดื่มแก้วแรก -- soft drink, น้ำดื่ม, เบียร์ไทย และ เหล้าไทย ) หลังจากนั้น เรามีการแสดงสดเพลงสไตล์ Electronic 3 การแสดงที่ยอดเยี่ยม
+ประกอบด้วย
+- Kijjaz (กิจจ๊าซ) หนึ่งในนักเล่นคีย์บอร์ดที่ได้รับความสนใจมากที่สุด และมีความชอบส่วนตัวใน program music กับ microcontroller และจะมาสร้างการแสดงพิเศษเพื่อพวกเราโดยเฉพาะ 
+- DCNXTR (De connextor) วง Electronic ที่ดีที่สุดของประเทศไทย  -- เป็นทั้ง innovative และ เต้นได้ในเวลาเดียวกัน ซึ่งจะให้คุณพบกับประสบการณ์ที่ไม่เคยมีมาก่อน
+- Orbital XX หนึ่งในวงที่มีเพลง Electronic ที่มาแรงที่สุดในกรุงเทพ
+
+แน่นอนว่ามันจะมีสิ่งต่างๆที่ทำให้ทุกคนสนุกสนานและนอกจากนี้ สถานที่จัดงานยังเต็มไปด้วยสถานที่สำหรับการพบปะพูดคุยมากกว่าสี่ชั้น 
+มาพบกันที่สีลม สถานที่ที่ดีที่สุดของกรุงเทพและเพลิดเพลินไปกับวงดนตรีอินดี้ที่ดีที่สุดของไทย 
+
+Sure to be something for everyone to enjoy and the venue has plenty of places
```

Comment:
I thought I got this part translated too

Reply:
Quite likely, but I don't know where the Thai ends ... @vtno :eyes: